kubernetes/loculus/templates/silo-deployment.yaml

```diff
@@ -43,6 +43,25 @@ spec:
           volumeMounts:
             - name: lapis-silo-shared-data
               mountPath: /data
+          livenessProbe:
```

Comment:
> consider SILO "live" as soon as the /info endpoint responds something (including errors)

I doubt the current implementation does this - I think an error would manifest as unhealthiness. IMO for now we could not add the livenessProbe if we can't figure out how to do this. I think it would be better not to add it than to add it like this

Reply:
(thanks for taking on the PR!)

Reply:
True - even once the database is loaded argocd still shows SILO as degraded.

Reply:
ah ok now it looks ok - it just seems a bit delayed

Reply:
Yeah but my point still stands (I think?)

<img width="578" alt="image" src="https://github.com/user-attachments/assets/48ae498c-9217-4c61-a478-226b46f7486c" />

Reply:
To be clear, I'm happy with the readinessProbe you've implemented!

Reply:
Cool! Was just going to suggest we only add the readiness probe. I'm not sure though - do we need the 
```
httpHeaders:
    - name: Accept
      value: application/json
```
part? will errors also be seen as errors for the readiness probe?

Reply:
Yeah I wasn't clear why that was added - I wouldn't bother unless you added it because something wasn't working

> will errors also be seen as errors for the readiness probe?

yes